public/scripts/votingController.js

```diff
@@ -0,0 +1,101 @@
+(function (module) {
+  var voteRepos = {};
+  var member;
+  var voteHistory;
+  var votesArray = ['S J RES 23', 'H R 1927', 'H R 2130', 'H R 1644']; // Specific bills we want to store
```

Comment:
Stretch goal... Is there a place that you can get data from on the 'Most Notable' bills that are currently active, or inactive but considered historic?